packages/itwinui-css/backstop/tests/slider.html

```diff
@@ -382,17 +463,444 @@ <h2>Ranges</h2>
             </output>
             <div class="iui-slider-ticks">
               <!-- You could generate the ticks based on your min, max & step values. -->
-              <span class="iui-slider-tick">0</span>
-              <span class="iui-slider-tick">10</span>
-              <span class="iui-slider-tick">20</span>
-              <span class="iui-slider-tick">30</span>
-              <span class="iui-slider-tick">40</span>
-              <span class="iui-slider-tick">50</span>
-              <span class="iui-slider-tick">60</span>
-              <span class="iui-slider-tick">70</span>
-              <span class="iui-slider-tick">80</span>
-              <span class="iui-slider-tick">90</span>
-              <span class="iui-slider-tick">100</span>
+              <div class="iui-slider-tick">
```

Comment:
Why tick bacame wrapper and now has two elements inside?

Reply:
This was done since in the old code for the horizontal slider, the number would go below the tick because the number could not fit in the iui-slider-tick. However, in the vertical slider, the ticks needed to be horizontal. So when the number would overflow out of the iui-slider-tick, the number would still go below the tick (as below).

Actual
<img width="60rem" src="https://user-images.githubusercontent.com/45748283/180241243-1179eadf-330f-44fc-9a07-fee7fd9678d8.png"/>

Required
<img width="60rem" src="https://user-images.githubusercontent.com/45748283/180241885-553839ba-2734-4142-ae24-5782560c592d.png"/>

I was not sure how to use the same HTML code to achieve it. So I thought to use flex on the iui-slider-tick and separate it into tick-mark and tick-number that are arranged in a row flex

Old code for a tick

```html
<div class="iui-slider-ticks">
  <span class="iui-slider-tick">0</span>
</div>
```

New code for a tick

```html
<div class="iui-slider-ticks">
  <div class="iui-slider-tick">
    <span class="iui-slider-tick-mark"></span><span class="iui-slider-tick-number">10</span>
  </div>
</div>
```

Reply:
It seems previously it was done in a hacky way, so I guess it is ok.